keyboards/mokey/mokey64/info.json

```diff
@@ -0,0 +1,81 @@
+{
+    "keyboard_name": "Mokey64",
+    "url": "",
+    "maintainer": "mokey",
+    "width": 15,
+    "height": 5,
```

Comment:
```suggestion
```

Reply:
Thank you very much for pointing out the shortcomings and making corresponding corrections. Hope to be reviewed and merged